zebrad/tests/acceptance.rs

```diff
@@ -62,7 +65,7 @@ fn generate_no_args() -> Result<()> {
     zebra_test::init();
     let (tempdir, _guard) = tempdir(ConfigMode::Ephemeral)?;
 
-    let child = get_child(&["generate"], &tempdir)?;
+    let child = get_child(&["generate"], &tempdir, ConfigMode::Ephemeral)?;
```

Comment:
Instead of duplicating the ConfigMode argument and passing it into `tempdir` and `get_child` can we check to see if `zebrad.toml` already exists in the test directory in `get_child`? This should also clean-up the diff for this PR a lot.

Reply:
Thanks, applied this at https://github.com/ZcashFoundation/zebra/pull/946/commits/6541ae77c550eb7c248a5aeb8958bb4bba4d545c.